typescript/raven.d.ts

```diff
@@ -205,21 +205,14 @@ declare module Raven {
         /** Log a breadcrumb */
         captureBreadcrumb(crumb: Breadcrumb): RavenStatic;
 
-        /**
-         * Clear the user context, removing the user data that would be sent to Sentry.
-         */
-        setUserContext(): RavenStatic;
-
         /*
-        * Set a user to be sent along with the payload.
+        * Set/Clear a user to be sent along with the payload.
         *
         * @param {object} user An object representing user data [optional]
         * @return {Raven}
         */
-        setUserContext(user: {
-            id?: string;
-            username?: string;
-            email?: string;
+        setUserContext(user?: {
```

Comment:
Isn't this the same as just setUserContext(user?: Object)? I'm not very familiar with TS.